### DISCUSSION TITLE
Refusal to load codespace

### DISCUSSION THREAD

**Discussion by JustusCSmith** [community/community#46878] [Codespaces]
### Select Topic Area

Product Feedback

### Body

I'm having an issue in Google Chrome where it will not load my code space in browser no matter what I do. I followed the troubleshooting guide in here to no avail. I wonder if there are any other suggested remedies. The same seems to apply to JupyterLab. I can only access my codespace via VS Code currently (which is not convenient for me or preferred).

**Comment by eljog** [comment]
Hi @JustusCSmith 
Sounds like you are able to use the codespace in VSCode desktop, but it does not open in webbrowser/jupypter. To better understand the issue, could you please share the error messages/screen shots? Do you see any errors in the browser console?

**Reply by JustusCSmith** [reply]
![image](https://user-images.githubusercontent.com/124707616/217906384-51701f49-cb84-4c49-9981-218eb84dfb62.png)
I see this in the top left of the browser window, but it never loads said codespace.
